packages/app/src/composables/useTransaction.ts

```diff
@@ -289,8 +288,8 @@ function mapTransfers(transfers: Api.Response.Transfer[]): TokenTransfer[] {
 }
 
 function sumAmounts(balanceChanges: TokenTransfer[]) {
-  const total = balanceChanges.reduce((acc, cur) => acc.add(cur.amount || 0), BigNumber.from(0));
-  return total.toHexString() as Hash;
+  const total = balanceChanges.reduce((acc, cur) => acc + BigInt(cur.amount || 0), BigInt(0));
+  return `0x${total.toString(16)}` as Hash;
```

Comment:
Same

Reply:
Added an util function and used it in places like this.